src/View/Helper/FormHelper.php

```diff
@@ -114,29 +128,15 @@ public function input($fieldName, array $options = [])
         switch ($options['type']) {
             case 'checkbox':
             case 'radio':
-                $this->templates(['label' => '{{text}}']);
-
                 if (!isset($options['inline'])) {
-                    $options['inline'] = $this->checkClasses('checkbox-inline', (array)$options['label'])
-                        || $this->checkClasses('radio-inline', (array)$options['label']);
-
-                    if (!$this->_align) {
-                        $this->templates([
-                            'checkboxContainer' => '<div class="checkbox">{{content}}{{help}}</div>',
-                            'checkboxContainerError' => '<div class="checkbox has-error">{{content}}{{error}}{{help}}</div>',
-                        ]);
-                    }
+                    $options['inline'] = $this->checkClasses($options['type'] . '-inline', (array)$options['label']);
                 }
 
                 if ($options['inline']) {
-                    $options['label'] = $this->injectClasses('checkbox-inline', (array)$options['label']);
-                    $this->templates(['inputContainer' => '{{content}}']);
+                    $options['label'] = $this->injectClasses($options['type'] . '-inline', (array)$options['label']);
                 }
 
-                unset($options['inline']);
-
                 break;
-
             case 'select':
                 if (isset($options['multiple']) && $options['multiple'] === 'checkbox') {
                     $this->templates(['checkboxWrapper' => '<div class="checkbox">{{label}}</div>']);
```

Comment:
I'd like to get rid of this also. The template reset in this method is only required because of this change that hasn't been moved yet.